src/config.js

```diff
@@ -4,24 +4,30 @@ const config = require('yargs')
     .describe('d', 'File location of device list (must end with .json).')
     .describe('g', 'Gateway password (to enable gateway light change)')
     .describe('h', 'Show this help')
-    .describe('l', 'possiblevalues: "error", "warn","info","debug"')
+    .describe('l', 'Logging level')
     .describe('m', 'mqtt broker url. See https://github.com/mqttjs/MQTT.js#connect-using-a-url')
+    .describe('k', 'accept self singed-certificates when using TLS. See https://github.com/mqttjs/MQTT.js#mqttclientstreambuilder-options')
     .describe('n', 'instance name. used as mqtt client id and as topic prefix')
+    .boolean('k')
     .alias({
       d: 'devices',
       g: 'password',
       h: 'help',
       l: 'logging',
       m: 'mqtt',
+      k: 'insecure',
       n: 'name'
     })
+    .choices('l', ['error', 'warn', 'info', 'debug'])
     .default({
       l: 'info',
       m: 'mqtt://127.0.0.1',
+      k: false,
       n: 'xiaomi'
     })
     .version()
     .help('help')
+    .wrap(null)
```

Comment:
What does this row do?

Reply:
that disables yargs default usage output (which is 80 columns) basically demaning `-h` usage output wrapping to the terminal. (ref doc .wrap )[https://github.com/yargs/yargs/blob/HEAD/docs/api.md#wrapcolumns]. It's purely cosmetic, unfortunately I couldn't find a way to wrap properly by using something like `.wrap(yargs.terminalWidth())` as suggested in the docs.